releasenotes.txt

```diff
@@ -0,0 +1,314 @@
+BISDN Linux
+4.8.0 Release Notes
+Date: 2022-09-05
```

Comment:
The images will be ready at the earliest day after tomorrow
```suggestion
Date: 2022-09-07
```

Reply:
With our current pipelines, the first images will be published some time tomorrow and the second set will be published on Wednesday. Instead of adapting to two different build dates, i think it's better to define the release day as today, since we start the first pipelines today and there will be no more changes merged.

Reply:
We can do that starting with the *next* release, but until now we always had this as the estimated date when the builds were finished, not when they were started.

Reply:
(consistent in terms of sticking to the release date as defined on our board and during the planning)

Reply:
Let's stick to how we did it the last few releases, and not change the way we do releases during a release.